samples/dashboards/component-options/events/demo.js

```diff
@@ -0,0 +1,106 @@
+const CSVConnector = Dashboards.DataConnector.types.CSV;
+const csvData = document.getElementById('csv').innerText;
+
+const connector = new CSVConnector({
```

Comment:
Instead of creating `connector` as you did, please use the `dataPool` as we did here: https://github.com/highcharts/highcharts/blob/dashboards/main/samples/dashboards/demos/minimal/demo.js

Then refer to the connect by it's name
```JS
        connector: {
            name: 'Vitamin'
        },
```

Reply:
Done